projects/plugins/jetpack/extensions/shared/memberships.js

```diff
@@ -112,18 +112,8 @@ const updateQueryStringParameter = function ( uri, key, value ) {
 export const setPurchaseResultCookie = function ( premiumContentJWTToken ) {
 	// We will set this in a cookie  - just in case. This will be reloaded in the refresh, when user clicks OK.
 	// But user can close the browser window before clicking OK. IN that case, we want to leave a cookie behind.
-	const hostname = window.location.hostname;
-	const domain = '.' + hostname;
-
 	document.cookie =
-		'wp-jp-premium-content-session' +
-		'=' +
-		premiumContentJWTToken +
-		'; expires=' +
-		0 +
-		'; path=/' +
-		'; domain=' +
-		domain;
+		'wp-jp-premium-content-session' + '=' + premiumContentJWTToken + '; expires=' + 0 + '; path=/';
```

Comment:
Since we're editing that string, what do you think about using template literals? It may make the whole thing a bit easier to read.

```suggestion
		`wp-jp-premium-content-session=${ premiumContentJWTToken }; expires=0; path=/`;
```

Reply:
Brilliant idea, thank you 😄

Reply:
I continue this work here https://github.com/Automattic/jetpack/pull/35849 and this is probably a PR that will be merged. Applied it there.